experimental/packages/opentelemetry-instrumentation-http/test/functionals/http-enable.test.ts

```diff
@@ -908,4 +908,87 @@ describe('HttpInstrumentation', () => {
       });
     });
   });
+
+  describe('capturing headers as span attributes', () => {
+    beforeEach(() => {
+      memoryExporter.reset();
+    });
+
+    before(() => {
+      instrumentation.setConfig({
+        headersToSpanAttributes: {
+          client: { requestHeaders: ['X-Client-Header1'], responseHeaders: ['X-Server-Header1'] },
+          server: { requestHeaders: ['X-Client-Header2'], responseHeaders: ['X-Server-Header2'] },
+        }
+      });
+      instrumentation.enable();
+      server = http.createServer((request, response) => {
+        response.setHeader('X-Server-Header1', 'server123');
```

Comment:
```suggestion
        response.setHeader('X-ServEr-hEeader1', 'server123');
```
or similar to verify that casing doesn't matter

Reply:
Thanks, added it!